tests/cloner_test.go

```diff
@@ -1265,6 +1266,29 @@ var _ = Describe("all clone tests", func() {
 				By("The clone should fail")
 				f.ExpectEvent(f.Namespace.Name).Should(ContainSubstring(controller.CloneValidationFailed))
 			})
+
+			It("Should not clone when PopulatedFor annotation exists", func() {
+				targetName := "target" + rand.String(12)
+
+				By(fmt.Sprintf("Creating target pvc: %s/%s", f.Namespace.Name, targetName))
+				targetPvc, err := utils.CreatePVCFromDefinition(f.K8sClient, f.Namespace.Name, utils.NewPVCDefinition(
+					targetName,
+					"1Gi",
+					map[string]string{
+						controller.AnnPodPhase:     "Succeeded",
```

Comment:
What happens if this annotation is missing?

Reply:
checked it, the test still passes, should I remove this annotation from the test?

Reply:
removed